### DISCUSSION TITLE
How to solve hidden field (like in html forms) requirement

### DISCUSSION THREAD

**Discussion by sezeryesiltas** [danvick/flutter_form_builder#866] [General]
Hidden fields are very comfortable in html forms.
I'm not sure but it might be good idea something like that:
`
FormBuilderHiddenField(
                name: 'id',
)
`

**Comment by danvick** [comment]
You could consider just using Flutter's `Visibility` widget

**Comment by anqit** [comment]
This would be cool, to be able to have keys in the form that might be computed and aren't necessarily from a widget in the widget tree
